src/pages/home/live-pricing/index.tsx

```diff
@@ -0,0 +1,158 @@
+
+    ${Header} {
+        font-weight: normal;
+        font-size: 2rem;
+        color: var(--color-grey-5);
+    }
+
+    ${({ selected }) =>
+        selected
+            ? css`
+                  border-bottom: 2px solid red;
+                  & ${Header} {
+                      color: var(--color-red);
+                  }
+              `
+            : css`
+                  border-bottom: 2px solid var(--color-grey-8);
+                  & ${Header} {
+                  }
+              `}
+`
+const DisclaimerText = styled(Header)`
+    color: var(--color-grey-5);
+    font-weight: normal;
+    text-align: center;
+    font-size: 1.6rem;
+`
+
+const LivePricing = () => {
+    const [selected_market, setSelectedMarket] = useState<TAvailableLiveMarkets>('forex')
+    const onMarketButtonClick = (selected) => {
+        setSelectedMarket(selected)
+    }
+    const [is_logged_in] = useAuthCheck()
+    const handleLogin = useHandleLogin()
+
+    return (
+        <LivePricingSection id="live-pricing">
+            <Markets>
+                <MarketsContainer>
+                    {market_buttons.map((marketItem) => (
+                        <>
+                            <MarketButton
+                                selected={marketItem.market_name === selected_market}
+                                key={marketItem.id}
+                                onClick={() => {
+                                    onMarketButtonClick(marketItem.market_name)
+                                }}
+                            >
+                                <Header type="paragraph-2">
+                                    <Localize translate_text={marketItem.button_text} />
+                                </Header>
+                            </MarketButton>
+                        </>
+                    ))}
+                </MarketsContainer>
+            </Markets>
+            <ContainerWrapper>
+                {market_buttons.map(
+                    (marketItem) =>
+                        marketItem.market_name === selected_market && (
+                            <Header
+                                type="paragraph-1"
+                                weight="normal"
+                                align="center"
+                                key={marketItem.id}
+                            >
+                                <Localize translate_text={marketItem.market_description} />
+                            </Header>
+                        ),
+                )}
+                <LiveMarketTable market={selected_market} />
+                <DisclaimerText>
+                    <Localize translate_text="All spreads are indicative. To view real-time spreads, please refer to your terminal." />
+                </DisclaimerText>
+                {is_logged_in ? (
+                    <Button onClick={handleRedirectToTradersHub} secondary>
+                        <Localize translate_text="Trade now" />
```

Comment:
Can we use new key extraction string approach from the beginning to prevent refactor in future?
```suggestion
                        <Localize translate_text="_t_Trade now_t_" />
```